packages/agents-activity/src/conversation/membershipSourceTypes.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright(c) Microsoft Corporation.All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { z } from 'zod'
+
+/**
+ * Enum defining the type of roster the user is a member of.
+ */
+export enum MembershipSourceTypes {
+  /**
+   * The user is a direct member of the current channel.
+   */
+  Channel = 'channel',
+
+  /**
+   * The user is a member of a team that is a member of the current channel.
```

Comment:
Doc update request after discussion with Anand:
```suggestion
   * The source is that of a team and the user is a member of that team.
```